sbi/sbi_types.py

```diff
@@ -30,9 +30,9 @@
 
 # Define alias types because otherwise, the documentation by mkdocs became very long and
 # made the website look ugly.
-TensorboardSummaryWriter = NewType("Writer", SummaryWriter)
+TensorboardSummaryWriter = NewType("TensorboardSummaryWriter", SummaryWriter)
 # TorchTransform = NewType("torch Transform", Transform)
-TorchModule = NewType("Module", Module)
+TorchModule = NewType("TorchModule", Module)
```

Comment:
what is the reason for these changes?

Reply:
OK, got it. After a closer look, I think this needs some refactoring. I will propose something and push a commit directly to this branch, OK? 
Have you enabled `maintainer` write access to this PR on your fork? (ticking the box on the right somewhere I believe).

Reply:
see #1637